src/opnsense/mvc/app/controllers/OPNsense/Routing/forms/dialogEditGateway.xml

```diff
@@ -58,17 +58,17 @@
     </field>
     <field>
         <id>gateway_item.monitor_killstates</id>
-        <label>Kill states when down</label>
+        <label>Failover States</label>
         <type>checkbox</type>
         <style>monitor_opt</style>
-        <help>When a monitor down event is triggered, kill all states to this gateway.</help>
+        <help>Move clients to another available working gateway when this gateway is down. This requires default gatway switching or an assigned gateway group to trigger.</help>
     </field>
     <field>
         <id>gateway_item.monitor_killstates_priority</id>
-        <label>Kill states metered</label>
+        <label>Failback States</label>
         <type>checkbox</type>
         <style>monitor_opt</style>
-        <help>When a monitor up event is triggered for a gateway with a higher priority (lower value) than this, kill all states to this gateway. Most common use for this switch are metered connections like LTE.</help>
+        <help>Move clients to a working gateway with a higher priority (lower value) than this gateway regardless of its monitor status. It requires default gatway switching or an assigned gateway group to trigger. Most common use for this otion are metered connections over LTE that should only be used when no other working gateway is available.</help>
```

Comment:
"regardless of its monitor status" confusion which gateway is meant: the current or the one that is supposed to be working regardless of its status ;)